ui.apps/src/main/content/jcr_root/apps/etoolbox-link-inspector/clientlibs/link-inspector-ui/js/console-ui.filter.js

```diff
@@ -18,12 +18,12 @@
  */
 (function (window, document, $, ELC, Granite, Coral) {
     'use strict';
-    var DIALOG_TITLE_LABEL = Granite.I18n.get('Filter Options');
-    var SUCCESS_DIALOG_TITLE_LABEL = Granite.I18n.get('Filter Options Applied');
-    var CANCEL_LABEL = Granite.I18n.get('Cancel');
-    var SUBMIT_FILTER_LABEL = Granite.I18n.get('Apply');
+    const DIALOG_TITLE_LABEL = Granite.I18n.get('Filter Options');
```

Comment:
Since the ES2015 syntax is utilized here, we probably need to add specific processor instructions to the clienlib definition. Something of this kind: https://github.com/exadel-inc/etoolbox-authoring-kit/blob/master/ui.apps/src/main/content/jcr_root/apps/etoolbox-authoring-kit/policies/.content.xml

Reply:
done